[2024-01-01T12:45:17.005+0000] {taskinstance.py:1957} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: snow_dag.is_snow_api_ready manual__2024-01-01T12:45:10.577350+00:00 [queued]>
[2024-01-01T12:45:17.020+0000] {taskinstance.py:1957} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: snow_dag.is_snow_api_ready manual__2024-01-01T12:45:10.577350+00:00 [queued]>
[2024-01-01T12:45:17.021+0000] {taskinstance.py:2171} INFO - Starting attempt 1 of 3
[2024-01-01T12:45:17.132+0000] {taskinstance.py:2192} INFO - Executing <Task(HttpSensor): is_snow_api_ready> on 2024-01-01 12:45:10.577350+00:00
[2024-01-01T12:45:17.137+0000] {standard_task_runner.py:60} INFO - Started process 848 to run task
[2024-01-01T12:45:17.140+0000] {standard_task_runner.py:87} INFO - Running: ['airflow', 'tasks', 'run', 'snow_dag', 'is_snow_api_ready', 'manual__2024-01-01T12:45:10.577350+00:00', '--job-id', '15', '--raw', '--subdir', 'DAGS_FOLDER/servicenow_dag.py', '--cfg-path', '/tmp/tmp4a_r_dxj']
[2024-01-01T12:45:17.143+0000] {standard_task_runner.py:88} INFO - Job 15: Subtask is_snow_api_ready
[2024-01-01T12:45:17.276+0000] {task_command.py:423} INFO - Running <TaskInstance: snow_dag.is_snow_api_ready manual__2024-01-01T12:45:10.577350+00:00 [running]> on host cdceb6054911
[2024-01-01T12:45:17.487+0000] {taskinstance.py:2481} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='prady1900' AIRFLOW_CTX_DAG_ID='snow_dag' AIRFLOW_CTX_TASK_ID='is_snow_api_ready' AIRFLOW_CTX_EXECUTION_DATE='2024-01-01T12:45:10.577350+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-01-01T12:45:10.577350+00:00'
[2024-01-01T12:45:17.488+0000] {http.py:130} INFO - Poking: api/now/table/incident
[2024-01-01T12:45:17.511+0000] {base.py:83} INFO - Using connection ID 'snow_api_id' for task execution.
[2024-01-01T12:45:21.365+0000] {base.py:293} INFO - Success criteria met. Exiting.
[2024-01-01T12:45:21.376+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=snow_dag, task_id=is_snow_api_ready, execution_date=20240101T124510, start_date=20240101T124517, end_date=20240101T124521
[2024-01-01T12:45:21.494+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2024-01-01T12:45:21.537+0000] {taskinstance.py:3281} INFO - 1 downstream tasks scheduled from follow-on schedule check
